feed.xml

```diff
@@ -0,0 +1,28 @@
+---
+---
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <title type="text" xml:lang="en">{{ site.name }}</title>
+  <link type="application/atom+xml" href="atom_feed_url" rel="self"/>
+  <link type="text/html" href="home_url_canonical" rel="alternate"/>
+  <updated>{{ site.time | date_to_xmlschema }}</updated>
+  <id>{{ site.url }}/feed.xml</id>
+
+  {% assign latest_events = site.data.conferences | reverse | slice: 0, 30 %}
+  {% for event in latest_events %}
+    {% if event.announced_at %}
+      <entry>
+        <published>{{ event.announced_at | date: "%Y-%m-%dT%H:%M:%S%z" }}</published>
```

Comment:
Could we do an `announced_on`? Dealing with timezones is annoying 😅  Or does the Atom standard expect a timestamp here? In that case, would just convert it to 00:00:00 UTC on that date. Any thoughts?

Reply:
Oh my bad, I somehow thought `event.announced_at` is a timestamp in the YAML given the `_at` suffix, but it's actually already a date, which is what I was alluding to.

But I guess the point then is if we want to rename the field to `announced_on`, since the convention usually is `_at` for timestamps and `_on` for dates.

Reply:
Good point. Renamed in https://github.com/ruby-conferences/ruby-conferences.github.io/pull/704/commits/fd8fe6d98415113250cf00d179937c98cb593fe8